tests/test_sequence_classification.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
 
 . ./tests/lib.sh
-
```

Comment:
100 `train_iterations` is unnecessary and will slow down the test. To get meaningful predictions ~20 iterations should be enough. Also `save_every`, `log_every`, and `val_every` can be set to half of `train_iterations` for this test.